.github/workflows/clt_nightly.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         image: [ "centos:7", "almalinux:8", "almalinux:9", "oraclelinux:9", "amazonlinux:latest" ]
     runs-on: ubuntu-22.04
     steps:
-      - uses: manticoresoftware/clt@0.1.13
```

Comment:
Because on a newer version in CI it fails the test - CLT test on Logstash and Filebeat - https://github.com/manticoresoftware/manticoresearch/actions/runs/7046770409/job/19180041403.